node/peer.js

```diff
@@ -340,12 +340,35 @@ TChannelPeer.prototype.countOutPending = function countOutPending() {
     return pending;
 };
 
+var QOS_UNCONNECTED = 0;
```

Comment:
nitpick: defaulting constants to strings is more readable and more debuggable.

I've found it hard to debug `req.state` and such because numbers.